controllers/shooter.py

```diff
@@ -1,91 +1,91 @@
-from enum import Enum
-
-from magicbot import StateMachine, default_state, state
+from magicbot import StateMachine, default_state, state, timed_state, will_reset_to
+from wpimath.geometry import Pose2d
 
+from components.chassis import Chassis
 from components.intake import Intake
 from components.shooter import Shooter
 from components.tilt import Tilt
 from components.turret import Turret
-
-
-class GoalHeight(Enum):
-    HIGH = 2
-    MID = 1
-    LOW = 0
-
+from utilities.ballistics import GoalHeight, calculate_ballistics
 
 # Setpoints for intaking state
 INTAKE_AZIMUTH: float
 INTAKE_TILT: float
```

Comment:
Are we planning to define and use these any time soon?

Reply:
No, I think they should be removed here, and set in the components themselves.